bgpd/bgp_nexthop.c

```diff
@@ -192,17 +192,10 @@ static void show_address_entry(struct hash_bucket *bucket, void *args)
 	struct listnode *node;
 	char str[INET6_ADDRSTRLEN] = {0};
 
-	if (addr->p->family == AF_INET) {
-		vty_out(vty, "addr: %s, count: %d : ", inet_ntop(AF_INET,
-				&(addr->p->u.prefix4),
-				str, INET_ADDRSTRLEN),
-				addr->ifp_name_list->count);
-	} else if (addr->p->family == AF_INET6) {
-		vty_out(vty, "addr: %s, count: %d : ", inet_ntop(AF_INET6,
-				&(addr->p->u.prefix6),
-				str, INET6_ADDRSTRLEN),
-				addr->ifp_name_list->count);
-	}
+	vty_out(vty, "addr: %s, count: %d : ",
+		inet_ntop(addr->p.family, &(addr->p.u.prefix),
+			  str, INET6_ADDRSTRLEN),
```

Comment:
not `sizeof(addr->p.u.prefix)` instead of `INET6_ADDRSTRLEN`?

Reply:
this is for the inet_ntop which needs str size.  Which is how big that buffer is.